src/platform_impl/windows/mod.rs

```diff
@@ -30,7 +30,7 @@ pub struct PlatformSpecificWindowBuilderAttributes {
     pub no_redirection_bitmap: bool,
     pub drag_and_drop: bool,
     pub skip_taskbar: bool,
-    pub class_name: String,
```

Comment:
Why is it not enough to use this existing functionality to override the class name?

Reply:
I'm not looking to change the class name (or the code that directly uses winit).

Also the result of registering the window class has to be checked to ensure memory safety.